libvast/src/plugin.cpp

```diff
@@ -33,6 +36,7 @@
 #include <dlfcn.h>
 #include <memory>
 #include <tuple>
+#include <unistd.h>
```

Comment:
I thought I needed those since the Mac CI run had a build error due to missing includes.. but now it works again so I removed them in https://github.com/tenzir/vast/pull/2935/commits/eaa93a43c9c711744a29093ad06818fbebe59520